aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -538,6 +538,7 @@ module aptos_framework::staking_contract {
         // In case there's any dust left, send them all to the staker.
         if (coin::value(&coins) > 0) {
```

Comment:
should this coin be exact the same as the pool's  pool.total_coins?  if so, should we just pool's total coins' amount?

Reply:
Yeah same number. We can use either. I don't think it matters much unless I'm overlooking something?